src/main/java/edu/ucla/library/iiif/auth/services/DatabaseServiceImpl.java

```diff
@@ -153,16 +137,9 @@ public DatabaseServiceImpl(final Vertx aVertx, final JsonObject aConfig) {
         return myDbConnectionPool.withConnection(connection -> {
             return connection.preparedQuery(UPSERT_DEGRADED_ALLOWED).execute(Tuple.of(aOrigin, aDegradedAllowed));
         }).recover(error -> {
-            return Future.failedFuture(LOGGER.getMessage(MessageCodes.AUTH_006, UPSERT_DEGRADED_ALLOWED, error));
+            return Future.failedFuture(new ServiceException(INTERNAL_ERROR, error.getMessage()));
         }).compose(upsert -> {
```

Comment:
Does `upsert` as a variable name still make sense here?

Reply:
I agree, would be good to align with our conventions elsewhere